examples/bare_etiss_processor/main.cpp

```diff
@@ -83,12 +84,17 @@ int main(int argc, const char *argv[])
     //        ETISS_SRCLOC);
     //  }
 
-    if (!dsys.loadRom(etiss::cfg().get<std::string>("sw_binary_rom", "").c_str()) ||
-        !dsys.loadRam(etiss::cfg().get<std::string>("sw_binary_ram", "").c_str()))
-    {
-        etiss::log(etiss::FATALERROR, "ram/rom not loaded properly\n");
+//     if (!dsys.loadRom(etiss::cfg().get<std::string>("sw_binary_rom", "").c_str()) ||
```

Comment:
should be removed instead of commented out. no code is lost since we use version control